src/tenants/planet/LeaderBoard/components/Stories.tsx

```diff
@@ -8,30 +8,40 @@ interface Props {}
 
 export default function Stories({}: Props): ReactElement {
   const { t, ready } = useTranslation(['planet']);
-  return ready ? (
-    <div className={styles.wrapper}>
-      <div className={styles.container}>
-        <div className={styles.storyCard}>
-          <img src={'/tenants/planet/images/leaderboard/restoreTrees.svg'} />
-          <div className={styles.storyContent}>
-            <h2>{t('planet:restoreTrees')}</h2>
-            <div>
-              <img src={'/tenants/planet/images/leaderboard/playIcon.svg'} />
-              <p>{t('planet:watchVideo')}</p>
-            </div>
+  return (
+    <>
+      {ready ? (
+        <div className={styles.wrapper}>
+          <div className={styles.container}>
+            <a className={styles.storyCard} href="https://a.plant-for-the-planet.org/principles">
```

Comment:
If we ever want to localize these pages, it would be better to link them like `https://a.plant-for-the-planet.org/${i18n.language}/principles` (needs i18n to get loaded) - at least this page is also available in German https://a.plant-for-the-planet.org/de/principles.

Reply:
(You can compare with my solution in https://github.com/Plant-for-the-Planet-org/planet-webapp/pull/1302.)